bumpversion/main.py

```diff
@@ -33,7 +33,7 @@ def echo(
 def _load_settings(ctx: click.Context, param: click.Option, value: str) -> str:
     """Load option defaults from config file."""
     # Use `construct` to skip pydantic validation.
-    settings = Settings.construct(config_file=value)  # type: ignore[call-arg]
+    settings = Settings(config_file=value)
```

Comment:
`construct` was used to allow display of `--help` even in cases configuration file was incorrect. I'd keep it as a fallback for cases `Settings` can't be initialized properly.

Reply:
This is tricky. The problem is that if we call `bumpversion` and ignore incorrect configuration file (which is what happens when we use `construct`), the user can actually get results that vastly differ from his expectations. `--help` doesn't do anything, but lot's of other use cases do.

Reply:
Example:
I call `bumpversion minor` with incorrect config. If I implement what you propose, config file will be ignored and bumpversion will bump the `minor` version with default `Settings` parameters. That can create quite a mess. I'm not inclined to make this change.

Reply:
Not great, not terrible.